frontend/src/index.jade

```diff
@@ -124,7 +124,11 @@ html
         .title
           .repoName {{ info.repo }}
           .author {{ info.name }} ({{ info.author }}) contribution from {{ info.minDate }} to {{ info.maxDate }}
-          .contribution(v-if="isLoaded") Total contribution: {{ totalLineCount }} lines
+          .contribution(v-if="isLoaded")
+            span Total LoC contributed:&nbsp;
+            span.loc {{ totalLineCount }}
+            br
+            span.fileType(v-if="totalLineCount > 0") &rarr; {{ filesLineInfo }}
```

Comment:
I was thinking you let vue do all the string handling (to keep all the diaplay logic to frontend), so you expose `filesLineObj` instead and v-for thru the keys.

```
span.filetype(...)
    template(v-for="type in Object.keys(filesLinesObj)")
        span {{ type }}: {{fileLinesObj[type] }}
```

Well, the idea is to remove most of the display logic from the model part of view, which is the data. Well just chnage as you seem fit.

Reply:
@ongspxm how do i ensure that the values are comma separated in the display using ```v-for```? if i do ``` span {{ type }} : {{ fileLinesObj[type] }} , ``` then I will have an extra `,` at the end of the line.

Reply:
Hahah, i would do it in css.


In the relevant part:

```
span.filetype{
    span::after{
        content: ', ';
    }
    
    span:last-child::after{
         content:'';
    }
}
```

Reply:
@ongspxm thanks for the suggestion! i've updated the implementation accordingly